distributed/client.py

```diff
@@ -4189,6 +4189,9 @@ def register_worker_plugin(self, plugin=None, name=None):
         name: str, optional
             A name for the plugin.
             Registering a plugin with the same name will have no effect.
+        **kwargs: optional
+            If you do pass an class name as the plugin, it will be instantiated
+            with any extra keyword arguments.
```

Comment:
A small suggestion on this parameter description, but otherwise this LGTM 

```suggestion
            If you pass a class as the plugin, instead of a class instance, then the
            class will be instantiated with any extra keyword arguments.
```

Reply:
Thanks!